src/main/java/com/cognifide/aemrules/checks/ContentResourceShouldBeNullCheckedCheck.java

```diff
@@ -0,0 +1,261 @@
+/*-
+ * #%L
+ * AEM Rules for SonarQube
+ * %%
+ * Copyright (C) 2015 Cognifide Limited
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.cognifide.aemrules.checks;
+
+import com.cognifide.aemrules.Constants;
+import com.cognifide.aemrules.tag.Tags;
+import java.util.HashMap;
+import java.util.Map;
+import org.sonar.check.Priority;
+import org.sonar.check.Rule;
+import org.sonar.plugins.java.api.JavaFileScanner;
+import org.sonar.plugins.java.api.JavaFileScannerContext;
+import org.sonar.plugins.java.api.tree.AssignmentExpressionTree;
+import org.sonar.plugins.java.api.tree.BaseTreeVisitor;
+import org.sonar.plugins.java.api.tree.ClassTree;
+import org.sonar.plugins.java.api.tree.IdentifierTree;
+import org.sonar.plugins.java.api.tree.IfStatementTree;
+import org.sonar.plugins.java.api.tree.MemberSelectExpressionTree;
+import org.sonar.plugins.java.api.tree.MethodInvocationTree;
+import org.sonar.plugins.java.api.tree.MethodTree;
+import org.sonar.plugins.java.api.tree.ReturnStatementTree;
+import org.sonar.plugins.java.api.tree.Tree.Kind;
+import org.sonar.plugins.java.api.tree.VariableTree;
+
+@Rule(
+    key = ContentResourceShouldBeNullCheckedCheck.RULE_KEY,
+    name = ContentResourceShouldBeNullCheckedCheck.RULE_MESSAGE,
+    priority = Priority.MINOR,
+    tags = Tags.AEM
+)
+public class ContentResourceShouldBeNullCheckedCheck extends BaseTreeVisitor implements
+    JavaFileScanner {
+
+    public static final String RULE_KEY = "AEM-18";
+
+    public static final String RULE_MESSAGE = "Always null check the returned value of Page.getContentResource() method";
+
+    public static final String GET_CONTENT_RESOURCE_METHOD = "getContentResource";
+
+    public static final String NOT_EQUAL = "NOT_EQUAL_TO";
```

Comment:
You can use `org.sonar.plugins.java.api.tree.Tree.Kind.NOT_EQUAL_TO`. Same applies for EQUAL_TO

Reply:
done